app/src/main/java/dev/arkbuilders/arkmemo/utils/SVGUtils.kt

```diff
@@ -0,0 +1,172 @@
+package dev.arkbuilders.arkmemo.utils
+
+import android.graphics.Color
+import android.graphics.Paint
+import android.graphics.Path as AndroidDrawPath
+import android.util.Log
+import android.util.Xml
+import dev.arkbuilders.arkmemo.ui.viewmodels.DrawPath
+import org.xmlpull.v1.XmlPullParser
+import java.nio.file.Path
+import java.util.Stack
+import kotlin.io.path.reader
+import kotlin.io.path.writer
+
+class SVG {
```

Comment:
Filename should match the only defined class. Also this class is pretty important so better move to another package, maybe create `graphics` package?